response_operations_ui/templates/social-view-case-details.html

```diff
@@ -16,41 +16,11 @@ <h1 class="jupiter u-mt-m" id="case_ref">{{attributes.Reference}}</h1>
         <section class="social-case-section u-mt-l">
             <h2 class="neptune">Property address</h2>
             <dl class="survey-info info-panel" id="case-attributes">
-                {% if attributes.Prem1 %}
-                <dt class="case-address__prem" id="case-prem1">
-                    {{attributes.Prem1}}
-                </dt>
-                {% endif %}
-                {% if attributes.Prem2 %}
-                <dt class="case-address__prem" id="case-prem2">
-                    {{attributes.Prem2}}
-                </dt>
-                {% endif %}
-                {% if attributes.Prem3 %}
-                <dt class="case-address__prem" id="case-prem3">
-                    {{attributes.Prem3}}
-                </dt>
-                {% endif %}
-                {% if attributes.Prem4 %}
-                <dt class="case-address__prem" id="case-prem4">
-                    {{attributes.Prem4}}
-                </dt>
-                {% endif %}
-                {% if attributes.District %}
-                <dt class="case-address__prem" id="case-district">
-                    {{attributes.District}}
-                </dt>
-                {% endif %}
-                {% if attributes.PostTown %}
-                <dt class="case-address__prem" id="case-post_town">
-                    {{attributes.PostTown}}
-                </dt>
-                {% endif %}
-                {% if attributes.Postcode %}
-                <dt class="case-address__prem postcode" id="case-postcode">
-                    {{attributes.Postcode}}
-                </dt>
-                {% endif %}
+            {% for name in displayed_attributes %}
+              <dt class="case-address__{{ name|lower }}" id="case-{{ name|lower|replace('_', '-') }}">
```

Comment:
This could be cleaned up a bit. Could you not add this logic to the Python code instead?

Reply:
This has already been cleaned up and (now) only has display logic in the template.

Reply:
Yeah this is a cleaned up version that @jcox-dev recommended, now we don't have to update the template with any new changes and just do them in the views 👍